sciplot/plot.hpp

```diff
@@ -42,334 +42,295 @@
 #include <sciplot/specs/ticspecs.hpp>
 #include <sciplot/util.hpp>
 
-namespace sciplot {
-
-using namespace internal;
-
-/// The class used to define the plots, show in a pop-up window and save to a file.
-class plot
+namespace sciplot
```

Comment:
What about no indentation inside `namespace`:

~~~cpp
namespace sciplot {
...
} // namespace sciplot

Reply:
Ok with me. I rely on auto-formatting, so I could check if I can prepare a .clang-format file that does that. Should I commit that so we can use it in the CI too?

Reply:
See 23664c8 to check if this is ok with you.

Reply:
This is all fine as long as we can slightly adapt it in the future if needed! ;)